ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch

```diff
@@ -270,6 +272,7 @@ index 0000000000..037467366e
 +    param->tune                     = svt_enc->tune;
 +    param->base_layer_switch_mode   = svt_enc->base_layer_switch_mode;
 +    param->qp                       = svt_enc->qp;
++   param->target_socket            = svt_enc->target_socket;
```

Comment:
could you keep the same Indent?

Reply:
Hi, @tianjunwork thanks for your reply. I'm dealing with build errors, will do indentation and white spaces thing for sure.
As for the default value in ffmpeg - our investigation shows that due to the NUMA issues  using all sockets (like in encoder library) makes performance worse than using a single socket unfortunately. That's why the socket default value in ffmpeg is zero currently (use socket zero) and it's not my change. I will keep it this way.

Reply:
@tianjunwork, I've fixed everything promised above, build works, so it would be great if you approve this PR ASAP - the corresponding socket selection functionality is in real use currently. 
As for the default target_socket value - please see https://github.com/OpenVisualCloud/SVT-VP9/blob/master/Source/Lib/Codec/EbEncHandle.c line 1696 where the line //config_ptr->target_socket = -1; has been commented (NOT by me :) causing the RIGHT behavior in FFmpeg defaulting it to zero